test/roundtrip/debug-names.txt

```diff
@@ -1,6 +1,10 @@
 ;;; TOOL: run-roundtrip
-;;; ARGS: --debug-names
+;;; ARGS: --debug-names --enable-bulk-memory
```

Comment:
Do we want a new test with this flag and the additions below? Otherwise we only test with this flag enabled.

Reply:
The reason I need to add this flag is that naming elem sections isn't possible without bulk memory enabled.   If we added another test it would looks just the same but I couldn't include the `elem` section name (its a parse error without bulk memory).

WDYT, should add a the second (module duplicate) test?  I'm happy to to if you think its worth it.

Reply:
Ic, I can't imagine many cases where the duplicate case will be useful, since bulk-memory flag shouldn't affect the way we handle names for other sections. LGTM to not add a test. Thanks for explaining!